package.json

```diff
@@ -83,30 +79,32 @@
     "semantic-release": "^24.0.0",
     "titleize": "^4.0.0",
     "ts-expect": "^1.3.0",
-    "typescript": "5.5.2"
+    "tslib": "^2.6.3",
+    "typescript": "5.5.2",
+    "typescript-eslint": "8.0.0-alpha.41"
   },
   "scripts": {
+    "bench": "npm run build && node test/bench.js",
+    "build": "npm run rollup && npm run build:types && npm run build:man",
+    "build:docs": "npm run build && node docs/build.js",
+    "build:man": "marked-man man/marked.1.md > man/marked.1",
+    "build:reset": "git checkout upstream/master lib/marked.cjs lib/marked.umd.js lib/marked.esm.js marked.min.js",
+    "build:types": "tsc && dts-bundle-generator --project tsconfig.json -o lib/marked.d.ts src/marked.ts && dts-bundle-generator --project tsconfig.json -o lib/marked.d.cts src/marked.ts",
+    "lint": "eslint --fix",
+    "rollup": "rollup -c rollup.config.js",
+    "rules": "node test/rules.js",
     "test": "npm run build && npm run test:specs && npm run test:unit",
-    "test:only": "npm run build && npm run test:specs:only && npm run test:unit:only",
     "test:all": "npm test && npm run test:umd && npm run test:types && npm run test:lint",
-    "test:unit": "node --test --test-reporter=spec test/unit/*.test.js",
-    "test:unit:only": "node --test --test-only --test-reporter=spec test/unit/*.test.js",
-    "test:specs": "node --test --test-reporter=spec test/run-spec-tests.js",
-    "test:specs:only": "node --test --test-only --test-reporter=spec test/run-spec-tests.js",
-    "test:lint": "eslint .",
+    "test:lint": "eslint",
+    "test:only": "npm run build && npm run test:specs:only && npm run test:unit:only",
     "test:redos": "node test/recheck.js > vuln.js",
+    "test:specs:only": "node --test --test-only --test-reporter=spec test/run-spec-tests.js",
+    "test:specs": "node --test --test-reporter=spec test/run-spec-tests.js",
     "test:types": "tsc --project tsconfig-type-test.json && attw -P --exclude-entrypoints ./bin/marked ./marked.min.js",
     "test:umd": "node test/umd-test.js",
-    "test:update": "node test/update-specs.js",
-    "rules": "node test/rules.js",
-    "bench": "npm run build && node test/bench.js",
-    "lint": "eslint --fix .",
-    "build:reset": "git checkout upstream/master lib/marked.cjs lib/marked.umd.js lib/marked.esm.js marked.min.js",
-    "build": "npm run rollup && npm run build:types && npm run build:man",
-    "build:docs": "npm run build && node docs/build.js",
-    "build:types": "tsc && dts-bundle-generator --project tsconfig.json -o lib/marked.d.ts src/marked.ts && dts-bundle-generator --project tsconfig.json -o lib/marked.d.cts src/marked.ts",
-    "build:man": "marked-man man/marked.1.md > man/marked.1",
-    "rollup": "rollup -c rollup.config.js"
```

Comment:
Whats changing in the scripts here? Is it just the order?

Reply:
Ya just reordering alphabetically